### ISSUE TITLE
Relation not Work and other issues

### ISSUE THREAD

**Issue by paranoiasystem** [SoftwareBrothers/admin-bro#127]
Hello I'am trying to use admin bro with mongoose, but the relation not work 

this is my two simple model

**menu.model.js**

```js
const mongoose = require('mongoose')

const { Schema } = mongoose

const MenuSchema = new Schema({
    name: String,
    pageId: {
        type: Schema.Types.ObjectId,
        ref: 'Page'
    }
})

const Menu = mongoose.model('Menu', MenuSchema)

module.exports = Menu
```

and this is **page.model.js**

```js
const mongoose = require('mongoose')

const { Schema } = mongoose

const PageSchema = new Schema({
  title: {
    type: String,
    required: true,
  },
  content: {
    type: String,
    required: true,
  },
  createdAt: Date,
})

const Page = mongoose.model('Page', PageSchema)

module.exports = Page
```

and the file **admin.router.js**

```js
const AdminBro = require('admin-bro')
const AdminBroExpress = require('admin-bro-expressjs')

AdminBro.registerAdapter(require('admin-bro-mongoose'))

const Menu = require('../models/menu.model')
const Page = require('../models/page.model')

const adminBro = new AdminBro({
    resources: [
        { 
            resource: Menu,
        },
        { 
            resource: Page,
        },
    ],
    rootPath: '/admin',
  })

const router = AdminBroExpress.buildRouter(adminBro)

module.exports = router
```
Page is already Created

![image](https://user-images.githubusercontent.com/9149503/63422908-89f91e80-c40b-11e9-9807-dced7383f79e.png)

but I never see here for make a relation
![image](https://user-images.githubusercontent.com/9149503/63422874-7c439900-c40b-11e9-9745-fceb77ec68f4.png)

---

And furing the use I receive a lot of errors like this:

 
```
TypeError: controller[route.action] is not a function
    at handler (C:\Users\xxxxxxxx\node_modules\admin-bro-expressjs\plugin.js:56:52)
    at Layer.handle [as handle_request] (C:\Users\xxxxxxxx\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\xxxxxxxx\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\xxxxxxxx\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\xxxxxxxx\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:275:10)
    at urlencodedParser (C:\Users\xxxxxxxx\node_modules\body-parser\lib\types\urlencoded.js:91:7)
    at Layer.handle [as handle_request] (C:\Users\xxxxxxxx\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\xxxxxxxx\node_modules\express\lib\router\index.js:335:12)

      Deprecation: this function will be removed in the next versions.
      Please use BaseAction.after hook instead
```

**Comment by paranoiasystem** [comment]
I think is an error in the api generate form admin bro

![image](https://user-images.githubusercontent.com/9149503/63428759-9768d580-c418-11e9-8885-af1127a2ced4.png)

**Comment by wojtek-krysiak** [comment]
the issue is related to admin-bro-expressjs. When you start writing something in the select box it should work. Working on that...

**Comment by wojtek-krysiak** [comment]
It should be fixed - please check out the version 1.0.1 of AdminBro.

**Comment by paranoiasystem** [comment]
@wojtek-krysiak yeah it now work, thanks for the fast fix
